<launch>
    <node pkg="socketcan_bridge" type="socketcan_to_topic_node" name="socketcan_to_topic_node" output="screen">
        <param name="can_device" value="can1"/>
    </node>
    <node pkg="can_receive" type="can_receive_node" name="can_receive_1" output="screen">
        <param name="can_device" value="can1"/>
        <param name="receiver_topic" type="string" value="/can1_raw"/>
    </node>
</launch>
